C9 test development

Phinhas214 opened the pull request:
# Description 
Added a test case C9 for the conversion procedure (from Fahrenheit to Widget). @Phinhas214, @aaronlui415, and @shesophast were working on this as new developers. 

Partially addresses #962 

## Type of change
- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.org/developer/pr/) ideas
- [x] I have removed text in ( ) from the issue request
- [x] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.org/developer/cla/) and each author is listed in the Description section.

## Limitations
Only addresses test case C9.

Commit:
new test feature branch

Commit:
C9 test draft

Commit:
🚀 solved C9

Co-authored-by: Sophia Alexander <sophiagrace23@g.ucla.edu>
Co-authored-by: Aaron Lui <aaronluii15@gmail.com>

Commit:
Merge branch 'development' into Phin_development
update branch

Commit:
🚀 added C9 test case
Co-authored-by: Sophia Alexander <sophiagrace23@g.ucla.edu>
Co-authored-by: Aaron Lui <aaronluii15@gmail.com>

Phinhas214 committed:
Merge branch 'OpenEnergyDashboard:development' into C9_test_development

Chocopepero commented:
Congrats to @Phinhas214, @aaronlui415, and @shesophast on submitting your first Pull Request! All students have signed the CLA. The test case looks good. Just need a final approval from @huss and it's good to merge.

huss committed:
make conversion reversed

I accidentally forgot to use the reversed conversion in the design doc.
This fixes the test case to do that. This is not the doing of the developers.

huss commented:
I made a mistake in the design document and did not reverse the conversion (used c2 instead of c6). I realized this during my review of this PR. I have pushed a commit to fix this since it was my fault. The original work was fine and correctly implemented what was then in the design document.

huss reviewed (approved):
@Chocopepero review is taken as approving. Everything looks good. Congratulations to  @Phinhas214, @aaronlui415 & @shesophast on their first accepted contribution to OED.
